### DISCUSSION TITLE
Mapbox Beta Migration Help

### DISCUSSION THREAD

**Discussion by ReduxMPX** [uber/react-map-gl#2251] [Q&A]
Mapbox just announced their new release of dynamic lighting and three dimensional building rendering,

They have a migration guide but this is inapplicable to applications using react-map-gl, 
how do I go about migrating the the V3 beta?

Links:
[Announcement URL](https://www.mapbox.com/blog/standard-core-style)
[Migration Guide](https://www.mapbox.com/blog/standard-core-style)

**Comment by vitalyts** [comment]
Hi, do you have any plans to support Mapbox GL JS v3?

[Migrate to Mapbox GL JS v3](https://docs.mapbox.com/mapbox-gl-js/guides/migrate-to-v3/)

**Reply by Pessimistress** [reply]
https://github.com/visgl/react-map-gl/issues/2254
